### ISSUE TITLE
Bug layout because of WebRTC card

### ISSUE THREAD

**Issue by besiktas97** [dwainscheeren/dwains-lovelace-dashboard#495]
When I add a entitie in the light section or in a area, in the most time I cant delete the one that I have made, because of a layout but. See screenshot below:

![image](https://user-images.githubusercontent.com/76499056/166082915-e490183a-0e42-42c1-bba9-7e9e46a5197e.png)

I have this issue on the mobile platform aswell on a laptop with Chrome/Edge browser.

Another thing is that I can enable, hidden or disable a entity for showing it in DD. When I want to hidden a entity, in this case a personcard, it still shows it in the area for tracking people. See screenshot below:

![image](https://user-images.githubusercontent.com/76499056/166083258-24aa9464-ab0d-4d28-baec-e433eb2a23b2.png)

![image](https://user-images.githubusercontent.com/76499056/166083263-c16a544f-4c76-4d73-9f84-f18413c97d23.jpeg)

**Comment by dwainscheeren** [comment]
Do you use the webrtc card? It causes issues in dd.

**Comment by besiktas97** [comment]
Yes I am using webrtc. At this moment I have removed the entity from the config file(s).

**Comment by dwainscheeren** [comment]
WebRTC is causing issues in DD. Please don't use it for now inside DD.

**Comment by besiktas97** [comment]
Its a must have for me to use it. For now until there is a fix I will delete or add entitys through the config files.

**Comment by besiktas97** [comment]
By the way about my second question. When I want to hide a entity or device its still remaining on the dashboard. Does this also a issue with webrtc?

**Comment by davros123** [comment]
Also a must have for me - any eta on a fix?
Is there a work around we could use to include it on an area page ? (I have is as a separate more page and that works fine).
rtsp is fast and has almost no lag - it's really very important for cameras.
Hoping you can fix this as I also just had this issue.

**Comment by besiktas97** [comment]
Will there be a fix for this one in the next update version?

**Comment by LordNex** [comment]
Mine was working well until a few days ago and now the whole screen is lost. Similar to the 3.2 issue a while back. My WebRTC card was playing fine but it totally hosed the layout or the ability to access the controls to modify the page. 

I checked my entities.yaml file and all the entities are there. But the main screen is gone. Anyone have an idea?  I'd hate to make a new ticket if this is an known issue with a work around.
